aten/src/ATen/xpu/XPUEvent.h

```diff
@@ -0,0 +1,145 @@
+#pragma once
+#include <ATen/xpu/XPUContext.h>
+#include <c10/util/Optional.h>
+
+namespace at::xpu {
+
+/*
+ * XPUEvent are movable not copyable wrappers around SYCL event. XPUEvent are
+ * constructed lazily when first recorded. It has a device, and this device is
+ * acquired from the first recording stream. Later streams that record the event
+ * must match the same device.
+ *
+ * Currently, XPUEvent does NOT support to export an inter-process event from
+ * another process via inter-process comunication(IPC). So it means that
+ * inter-process communication for event handles between different processes is
+ * not available. This could impact some applications that rely on cross-process
+ * synchronization and communication.
+ */
+struct TORCH_XPU_API XPUEvent {
+  // Constructors
+  XPUEvent() noexcept = default;
+  XPUEvent(bool enable_timing) noexcept : enable_timing_{enable_timing} {}
```

Comment:
Is there a benefit of having 2 constructors instead of one with default argument?
```suggestion
  XPUEvent(bool enable_timing = false) noexcept : enable_timing_{enable_timing} {}
```

Reply:
done. one with a default argument is better.